lib/tasks/preflight.ex

```diff
@@ -3,27 +3,9 @@ defmodule Mix.Tasks.Preflight do
 
   @shortdoc "Checks for necessary ENVARS"
   def run(_) do
-    check_envars()
```

Comment:
I'm not sure about the evaluation order of mix tasks, but it could be that we can switch this to check for the Mix config defining non-empty values, rather than deleting the check outright. Can you check on this to see if it's a viable approach?